README.md

```diff
@@ -279,24 +291,25 @@ import "C"
 import "unsafe"
 
 func main() {
-	path := C.CString("/tmp/go.zip")
-	zip := C.zip_open(path, 6, 'w')
+ path := C.CString("/tmp/go.zip")
+ zip := C.zip_open(path, 6, 'w')
 
-	entryname := C.CString("test")
-	C.zip_entry_open(zip, entryname)
+ entryname := C.CString("test")
+ C.zip_entry_open(zip, entryname)
 
-	content := "test content"
-	buf := unsafe.Pointer(C.CString(content))
-	bufsize := C.size_t(len(content))
-	C.zip_entry_write(zip, buf, bufsize)
+ content := "test content"
+ buf := unsafe.Pointer(C.CString(content))
+ bufsize := C.size_t(len(content))
+ C.zip_entry_write(zip, buf, bufsize)
 
-	C.zip_entry_close(zip)
+ C.zip_entry_close(zip)
 
-	C.zip_close(zip)
+ C.zip_close(zip)
 }
```

Comment:
This formatting is incorrect. I'm not saying the original one is good, but at least for C looks much better.
I would suggest to focus in this PR on source code formatting (to be consistent).

Reply:
Done that :)